packages/typespec-azure-portal-core/src/decorators.ts

```diff
@@ -0,0 +1,364 @@
+import { ArmResourceKind, getArmResourceKind } from "@azure-tools/typespec-azure-resource-manager";
+import {
+  CompilerHost,
+  DecoratorContext,
+  Model,
+  ModelProperty,
+  Program,
+  StringLiteral,
+  Type,
+  getDirectoryPath,
+  getSourceLocation,
+  normalizePath,
+  resolvePath,
+  validateDecoratorUniqueOnNode,
+} from "@typespec/compiler";
+import * as fs from "fs";
```

Comment:
remove this import and use the compiler host for any fs

Reply:
removed thank you!